app/src/main/res/drawable/ic_landscape_onboarding_2.xml

```diff
@@ -35,5 +35,5 @@
   </path>
   <path
       android:pathData="M202.2,157.8H167V105H114.2v52.8H79l61.6,61.6ZM79,237v17.6H202.2V237Z"
-      android:fillColor="#02655c"/>
+      android:fillColor="#00655c"/>
```

Comment:
Don't update this color code. I have changed the description of your issue and therefore its fine if its inconsistent right now.

Reply:
Done